### DISCUSSION TITLE
Success on Kali/Debian 6.8.11-amd64 with Trendnet TEW-809UB (RTL8814au)

### DISCUSSION THREAD

**Discussion by roadrunnerguy** [morrownr/8814au#136] [General]
Just a quick thank you and success report. Seems to work great for the things I've tried so far. Now I can finally start debugging some odd WiFi issues I've had forever. Different devices will connect to just 1 of 2 different routers - each device has its preference for which one. Vendors are unlikely to change anything, but I still want to know why they behave as they do. Between this driver and the Monitor_Mode scripts I can now start the debugging process. Thanks for the excellent tool!

**Comment by morrownr** [comment]
Glad it is working for you.

FYI: If all goes with current projects, we should start the work on an in-kernel standards compliant 8814au driver soon. I'll post at the top of the README when we are ready for testers... in case you are interested.
